packages/wonder-blocks-labeled-field/src/components/flexible-labeled-field.tsx

```diff
@@ -0,0 +1,67 @@
+import {View} from "@khanacademy/wonder-blocks-core";
+import {semanticColor, sizing} from "@khanacademy/wonder-blocks-tokens";
+import {BodyText} from "@khanacademy/wonder-blocks-typography";
+import {StyleSheet} from "aphrodite";
+import * as React from "react";
+
+type Props = {
+    id?: string;
+    testId?: string;
+    label: React.ReactNode;
+    field: React.ReactElement;
+    helperTextAbove?: React.ReactNode;
+    helperTextBelow?: React.ReactNode;
```

Comment:
What naming is preferred for this prop?
- `helperTextAbove`/`helperTextBelow`
- `helperTextBefore`/`helperTextAfter`
- `helperTextStart`/`helperTextEnd`

Something else?